projects/packages/my-jetpack/_inc/components/product-detail-card/style.module.scss

```diff
@@ -150,6 +150,13 @@
 	margin-bottom: calc( var( --spacing-base ) * 3 );
 }
 
+.price-note {
+	font-size: var(--font-body-extra-small);
+	color: var( --jp-gray-20 );
+	margin-top: calc( 0px - ( var( --spacing-base ) * 3 ) );
+	margin-bottom: calc( var( --spacing-base ) * 3 );
+}
+
```

Comment:
Since we took out the price note ("* Not eligible for introductory discount") text, we should probably remove this styling for it.